### PR TITLE
Checking master execution

### PR DESCRIPTION
**Pull request by dmarcas**
This branch remove two hardcoded sleep in locustTask:
-It implements two solutions, one per windows and onether for linux.
-The implementation needs jdk version 9 : needed to implement the solution for linux.
-Modified the docker file in order to adapt it to jdk 9.
-Minor changes: added ingored tag in runner, to add the possibility to ingore features.

### CODE REVIEW

File: src/main/java/performance/LocustOperations.java [hunk 1]
```diff
@@ -98,15 +99,41 @@ private void clearValues() {
...
     public void executePerformanceTask(DataTable testData) throws Exception {
-        this.setTestData(testData);
-        TimeUnit.SECONDS.sleep(5);
-        this.executeMaster();
-        TimeUnit.SECONDS.sleep(10);
-        this.setUpSlave();
-        this.executeTask(testData);
-        TimeUnit.MINUTES.sleep(this.testTime);
-        this.locust.stop();
-        this.clearValues();
```
Comment (therussiankid92) [inline src/main/java/performance/LocustOperations.java:109]: So, finally there was no need for something in Linux?

File: src/test/resources/features/Functional.feature [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Feature: Functional Test
```
Comment (therussiankid92) [inline src/test/resources/features/Functional.feature]: Can you remove the ignore? I want to see that both Functional and Non-Functional Scenario work in the pipeline

File: src/main/java/performance/LocustOperations.java (hunk 1 above)
Comment (dmarcas) [reply src/main/java/performance/LocustOperations.java:109]: Yes, you can find it in the line 120 in the green section...

File: src/test/resources/features/Functional.feature (hunk 2 above)
Comment (dmarcas) [reply src/test/resources/features/Functional.feature]: Sure, later I will remove the tag from the feature file...

**Review by therussiankid92 — APPROVED** [review]
Had some downtime during onboarding and made a commit.
Aproved

**Pull request merged**